corehq/apps/reports/filters/api.py

```diff
@@ -254,11 +257,11 @@ def utils(self):
     @property
     def data_sources(self):
         return [
-            (self.get_users_size, self.get_users),
+            (self.get_all_users_size, self.get_all_users),
```

Comment:
This one I think would also want to see only active users, as it's used to assign users to locations and groups.

Reply:
https://github.com/dimagi/commcare-hq/pull/21564/commits/21bf9b797b16c46704d66b1ca327103f39b88e9e